### PR TITLE
1019 create the Android stream encryption guide for ma

### PR DESCRIPTION
**Pull request by saudsami**

### CODE REVIEW

File: shared/media-accelerator/develop/_media-stream-encryption.mdx [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+## Understand the tech
+
+To ensure secure communication, your <Vpl k="CLIENT" /> uses an SSL [key](https://en.wikipedia.org/wiki/Key_authentication) and [salt](https://en.wikipedia.org/wiki/Salt_(cryptography)) to encrypt and decrypt data in the channel. You use the key and salt to create an encryption configuration. <Vg k="AGORA_BACKEND" /> uses the encryption configuration to encrypt a stream and sends it to remote users. When a remote user receives an encrypted media stream, the remote <Vpl k="CLIENT" /> decrypts it using the same salt and key.
+
+The following figure shows the call flow for media stream encryption:
+
+![Encrypt media stream](/images/common/media-stream-encryption.png)
+
+All users in a channel must use the same encryption configuration to initiate `agoraRtcService` and enable encryption before joining a channel. If you don’t have the correct configuration, you cannot decrypt channel content. Best practice is to have your authentication system generate a new key and salt regularly.
```
Comment (billy-the-fish) [inline shared/media-accelerator/develop/_media-stream-encryption.mdx:21]: must use the same encryption configuration. You set this up when you initiate the Agora Engine and enable ...

File: shared/media-accelerator/develop/encrypt-media-streams/reference/android.mdx [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<PlatformWrapper platform="android">
+
+### API reference
+
+- <Link to="{{Global.MA_SDK_API_REF_ANDROID}}/classio_1_1agora_1_1rtc_1_1_agora_rtc_service.html#a0fb86ff4db8c4e9ec7080503ac479cad">setParams</Link>
```
Comment (billy-the-fish) [inline shared/media-accelerator/develop/encrypt-media-streams/reference/android.mdx:5]: Just a reminder, we need to find the API ref so we can add it to the system.

**Review by billy-the-fish — CHANGES_REQUESTED** [review]
Nearly there. Good stuff.

File: shared/media-accelerator/develop/_media-stream-encryption.mdx (hunk 1 above)
Comment (saudsami) [reply shared/media-accelerator/develop/_media-stream-encryption.mdx]: Fixes

**Pull request merged**